### ISSUE TITLE
tests need to trigger once a PR is approved or at least once requested through build comment

### ISSUE THREAD

**Issue by ldemailly** [istio/istio#433]
Example : https://github.com/istio/istio/pull/381

Skipping [#381 Kubernetes RBAC - Allow ingress to read secrets for TLS ingresses] because of user restriction (user - GHUser@3dbb042[login=lachie83,location=<null>,blog=<null>,email=<null>,name=<null>,company=<null>,followers=0,following=0,url=https://api.github.com/users/lachie83,id=6912984])

same as https://github.com/istio/istio.github.io/issues/308

**Comment by ldemailly** [comment]
I know this may change with prow but it's blocking

**Comment by sebastienvas** [comment]
duplicate of https://github.com/istio/istio/issues/433

**Comment by ldemailly** [comment]
I opened the other one earlier but it was in the wrong component (web) while it applies all across istio but sure
